cmds/boot/uinit/main.go

```diff
@@ -50,6 +51,17 @@ func main() {
 		signal.Ignore()
```

Comment:
Should the watchdog be stopped before being able to drop into a shell?

Reply:
Yes otherwise after dropping into the shell the watchdog will keep running. I will move the code up and update the PR later.

Reply:
ok but please let's make sure to get this code in with the change, it's very much needed :-)